Python/Tests/Analysis/LanguageServerTests.cs

```diff
@@ -309,6 +309,44 @@ class MyClass:
             );
         }
 
+        [TestMethod, Priority(0)]
+        public async Task SignatureHelp() {
+            var s = await CreateServer(null);
+            var mod = await AddModule(s, @"f()
+def f(): pass
+def f(a): pass
+def f(a, b): pass
+def f(a, *b): pass
+def f(a, **b): pass
+def f(a, *b, **c): pass
+
+");
+
+            await AssertSignature(s, mod, 0, 2,
+                new string[] { "f()", "f(a:=)", "f(a:=,b:=)", "f(a:=,*b:tuple=)", "f(a:=,**b:dict=)", "f(a:=,*b:tuple=,**c:dict=)" },
+                new string[0]
+            );
+
+            if (Default.Configuration.Version.Major != 3) {
+                return;
+            }
+
+            await s.UnloadFileAsync(mod);
+
+            mod = await AddModule(s, @"f()
+def f(a : int): pass
+def f(a : int, b: int): pass
+def f(x : str, y: str): pass
+def f(a = 2, b): pass
+
+");
+
+            await AssertSignature(s, mod, 0, 2,
+                new string[] { "f(a:int=)", "f(a:int=2,b:int=)", "f(x:str=,y:str=)" },
+                new string[0]
+            );
```

Comment:
Do you need to call `await s.UnloadFileAsync(mod);` at the end of test?

Reply:
No, we throw the server away completely.